src/ophys_etl/workflows/pipeline_module.py

```diff
@@ -62,12 +63,19 @@ def __init__(
         self._ophys_container = ophys_container
         self._docker_tag = docker_tag
         self._now = datetime.datetime.now()
-
         os.makedirs(self.output_path, exist_ok=True)
 
         if prevent_file_overwrites:
             self._validate_file_overwrite()
 
+        if isinstance(self.module_schema, DefaultSchema):
+            self.inputs = self.module_schema.load(data=self.module_args)
```

Comment:
It was previously a public property. So I should change it to private? 
I would need to find all the instances of where it was used and change them all the private. Given the commonality of the string "input", it may be difficult to find all the instances and change it to private.

Reply:
There is a difference between when you do `self.inputs` vs have a property called `inputs`. The latter does not allow setting a value which is what we want and is the correct pattern. So I am asking is to set `self._inputs` and then add a property 
```
@property
def inputs(self):
    return self._inputs
```